packages/sources/enzyme/src/config.ts

```diff
@@ -0,0 +1,23 @@
+import { Requester, util } from '@chainlink/ea-bootstrap'
+import { Config as BaseConfig } from '@chainlink/types'
+import { ethers } from 'ethers'
+
+export const NAME = 'ENZYME'
+
+export const ENV_RPC_URL = 'RPC_URL'
+
+export const DEFAULT_ENDPOINT = 'calcnav'
```

Comment:
Shouldn't this be `calcNav` according to the `README`?

Reply:
Thanks for the heads up. I made the endpoint name case consistent everywhere now.